impeller/renderer/renderer_dart_unittests.cc

```diff
@@ -72,22 +79,16 @@ INSTANTIATE_PLAYGROUND_SUITE(RendererDartTest);
 
 TEST_P(RendererDartTest, CanRunDartInPlaygroundFrame) {
   auto isolate = GetIsolate();
-
-  SinglePassCallback callback = [&](RenderPass& pass) {
```

Comment:
Any reason not to retain this test, and add the new one as a new test?

Reply:
Not really, added it back.